app/views/users/show.html.erb

```diff
@@ -0,0 +1,45 @@
+<h1 class="text-center"><%= @user.name %></h1>
+<div class="container">
+<div class="text-center">
+
+  <h4><strong> Events You Created </strong></h4>
+<ol class="list-unstyled">
+  <% @creators_events.each do |event| %>
+      <li>
+        <%= link_to event.name, event %>
+      </li>
+  <% end %>
+</ol>
+<hr>  
+  <h5><strong> Past Events </strong></h5>
+  <ol class="list-unstyled">
+    <% @past_user_events.each do |event| %>
+      <li>
+        <%= link_to event.name, event %>
+      </li>
+    <% end %>
+  </ol>
+  <hr>
+  <h5><strong> Upcoming Events To Attend </strong></h5>
+
+<ol class="list-unstyled">
+  <% @upcoming_user_events.each do |event| %>
+      <li>
+        <%= link_to event.name, event %>
+      </li>
+  <% end %>
+</ol>
+</div>
+<hr>
+  <% if actual_user? %>
+  <ul class="nav d-flex justify-content-center mt-5"> 
+  <li>
+    <%= link_to 'Edit User Account', edit_user_path(@user), class: 'btn btn-light' %>  
+  </li>
+  <li>
+    <%= link_to 'Delete Account', user_path(@user), method: :delete, class: 'btn btn-light ml-2'  %>
+  </li>
+  </ul>
+  
+<% end  %>
```

Comment:
- [ ] create a helper method that has the if/else logic and render the markup within the helper method (use a partial when the condition has several lines) instead of doing it straight in the view